lib/testutils/watch.go

```diff
@@ -0,0 +1,146 @@
+package testutils
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
do you really need `ginkgo` here? seems like you're only doing `gomega` assertions